### PR TITLE
Add sbt configuration to publish on Teads Nexus

### PR DESCRIPTION
**Pull request by Elyrixia**

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1"
```
Comment (pdalpra) [inline version.sbt]: `0.1-SNAPSHOT` :)

Comment (Elyrixia) [inline version.sbt]: I thought we should publish the first version as 0.1 directly and then change it to snapshot ?

Comment (pdalpra) [inline version.sbt]: Save it as snapshot, sbt-release will change the version to the release version you'll chose then put it back to the next snapshot.
As is, next run of `publish` would publish a release.

Comment (Elyrixia) [inline version.sbt]: Ok I should have read the sbt-release README before pushing/saying anything, sorry !

**Comment by AlixBa** [comment]
lgty @pdalpra?

File: version.sbt (hunk 1 above)
Comment (pdalpra) [inline version.sbt]: no problem :)

**Comment by pdalpra** [comment]
LGTM :boom:

**Pull request merged**